daily_open_prs.rb

```diff
@@ -14,29 +16,29 @@
   opts.on('-n', '--namespace NAME', 'GitHub namespace. Required.') { |v| options[:namespace] = v }
   opts.on('-r', '--repo-regex REGEX', 'Repository regex') { |v| options[:repo_regex] = v }
```

Comment:
Are these parameters used ? Once the modules.json has been defined it already contains the namespace